From 91c95d63d6f046faad1f0ea0c20a6d43793f572f Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Mon, 15 Apr 2024 09:41:26 +0200
Subject: [PATCH 0765/1695] treewide: stm32mp2: fix PMIC2 st property names

Fix PMIC2 st property names.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f8eb9828e919d789045b7f12547547b0a3ed14b6 treewide: stm32mp2: fix PMIC2 st property names]
Change-Id: Ie0ce73aa8a050f2e2ecc50652c4b2e2ccad14613
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/373459
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp235f-dk.dts          | 48 +++++++++----------
 core/arch/arm/dts/stm32mp257f-dk.dts          | 48 +++++++++----------
 core/arch/arm/dts/stm32mp257f-ev1.dts         | 48 +++++++++----------
 .../arm/plat-stm32mp2/drivers/stm32mp2_pmic.c |  6 +--
 .../devicetree/bindings/mfd/st,stpmic2.yaml   | 10 ++--
 5 files changed, 80 insertions(+), 80 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp235f-dk.dts b/core/arch/arm/dts/stm32mp235f-dk.dts
index 88a34eef1..dec4888d3 100644
--- a/core/arch/arm/dts/stm32mp235f-dk.dts
+++ b/core/arch/arm/dts/stm32mp235f-dk.dts
@@ -86,8 +86,8 @@
 				regulator-min-microvolt = <800000>;
 				regulator-max-microvolt = <910000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <2>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <2>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-off-in-suspend;
@@ -107,8 +107,8 @@
 				regulator-min-microvolt = <820000>;
 				regulator-max-microvolt = <820000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -137,8 +137,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-always-on;
 				st,mask-reset;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -163,8 +163,8 @@
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -186,8 +186,8 @@
 				regulator-min-microvolt = <1100000>;
 				regulator-max-microvolt = <1100000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -211,8 +211,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-always-on;
 				regulator-over-current-protection;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -242,16 +242,16 @@
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
 				regulator-over-current-protection;
-				st,pwrctrl_sel = <3>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <3>;
+				st,pwrctrl-reset;
 			};
 			vdd1_ddr: ldo3 {
 				regulator-name = "vdd1_ddr";
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -281,8 +281,8 @@
 				regulator-min-microvolt = <5000000>;
 				regulator-max-microvolt = <5000000>;
 				st,regulator-bypass-microvolt = <5000000>;
-				st,pwrctrl_sel = <0>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <0>;
+				st,pwrctrl-enable;
 			};
 			v5v_vconn: ldo6 {
 				regulator-name = "v5v_vconn";
@@ -290,8 +290,8 @@
 				regulator-max-microvolt = <4000000>;
 				regulator-always-on;
 				st,regulator-bypass-microvolt = <4000000>;
-				st,pwrctrl_sel = <0>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <0>;
+				st,pwrctrl-reset;
 			};
 			vdd_sdcard: ldo7 {
 				regulator-name = "vdd_sdcard";
@@ -299,8 +299,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-over-current-protection;
 				st,regulator-bypass-microvolt = <3300000>;
-				st,pwrctrl_sel = <3>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <3>;
+				st,pwrctrl-reset;
 			};
 			vddio_sdcard: ldo8 {
 				regulator-name = "vddio_sdcard";
@@ -308,8 +308,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-over-current-protection;
 				st,regulator-bypass-microvolt = <3300000>;
-				st,pwrctrl_sel = <3>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <3>;
+				st,pwrctrl-reset;
 			};
 		};
 	};
diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index 33ca16a60..d1c4b947a 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -90,8 +90,8 @@
 				regulator-min-microvolt = <800000>;
 				regulator-max-microvolt = <910000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <2>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <2>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-off-in-suspend;
@@ -111,8 +111,8 @@
 				regulator-min-microvolt = <820000>;
 				regulator-max-microvolt = <820000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -141,8 +141,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-always-on;
 				st,mask-reset;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -167,8 +167,8 @@
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -190,8 +190,8 @@
 				regulator-min-microvolt = <1100000>;
 				regulator-max-microvolt = <1100000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -215,8 +215,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-always-on;
 				regulator-over-current-protection;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -246,16 +246,16 @@
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
 				regulator-over-current-protection;
-				st,pwrctrl_sel = <3>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <3>;
+				st,pwrctrl-reset;
 			};
 			vdd1_ddr: ldo3 {
 				regulator-name = "vdd1_ddr";
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -285,8 +285,8 @@
 				regulator-min-microvolt = <5000000>;
 				regulator-max-microvolt = <5000000>;
 				st,regulator-bypass-microvolt = <5000000>;
-				st,pwrctrl_sel = <0>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <0>;
+				st,pwrctrl-enable;
 			};
 			v5v_vconn: ldo6 {
 				regulator-name = "v5v_vconn";
@@ -294,8 +294,8 @@
 				regulator-max-microvolt = <4000000>;
 				regulator-always-on;
 				st,regulator-bypass-microvolt = <4000000>;
-				st,pwrctrl_sel = <0>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <0>;
+				st,pwrctrl-reset;
 			};
 			vdd_sdcard: ldo7 {
 				regulator-name = "vdd_sdcard";
@@ -303,8 +303,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-over-current-protection;
 				st,regulator-bypass-microvolt = <3300000>;
-				st,pwrctrl_sel = <3>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <3>;
+				st,pwrctrl-reset;
 			};
 			vddio_sdcard: ldo8 {
 				regulator-name = "vddio_sdcard";
@@ -312,8 +312,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-over-current-protection;
 				st,regulator-bypass-microvolt = <3300000>;
-				st,pwrctrl_sel = <3>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <3>;
+				st,pwrctrl-reset;
 			};
 		};
 	};
diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index a541c868c..e2c8c5eed 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -90,8 +90,8 @@
 				regulator-min-microvolt = <800000>;
 				regulator-max-microvolt = <910000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <2>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <2>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-off-in-suspend;
@@ -111,8 +111,8 @@
 				regulator-min-microvolt = <820000>;
 				regulator-max-microvolt = <820000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -141,8 +141,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-always-on;
 				st,mask-reset;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -167,8 +167,8 @@
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -191,8 +191,8 @@
 				regulator-max-microvolt = <1200000>;
 				regulator-always-on;
 				regulator-pull-down;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -216,8 +216,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-always-on;
 				regulator-over-current-protection;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -247,14 +247,14 @@
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
 				regulator-over-current-protection;
-				st,pwrctrl_sel = <3>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <3>;
+				st,pwrctrl-reset;
 			};
 			vtt_ddr: ldo3 {
 				regulator-name = "vtt_ddr";
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-off-in-suspend;
@@ -281,8 +281,8 @@
 				regulator-min-microvolt = <2500000>;
 				regulator-max-microvolt = <2500000>;
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
@@ -306,8 +306,8 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-over-current-protection;
 				st,regulator-bypass-microvolt = <3300000>;
-				st,pwrctrl_sel = <3>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <3>;
+				st,pwrctrl-reset;
 			};
 			vddio_sdcard: ldo8 {
 				regulator-name = "vddio_sdcard";
@@ -315,14 +315,14 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-over-current-protection;
 				st,regulator-bypass-microvolt = <3300000>;
-				st,pwrctrl_sel = <3>;
-				st,pwrctrl_reset;
+				st,pwrctrl-sel = <3>;
+				st,pwrctrl-reset;
 			};
 			vref_ddr: refddr {
 				regulator-name = "vref_ddr";
 				regulator-always-on;
-				st,pwrctrl_sel = <1>;
-				st,pwrctrl_enable;
+				st,pwrctrl-sel = <1>;
+				st,pwrctrl-enable;
 
 				default {
 					regulator-on-in-suspend;
diff --git a/core/arch/arm/plat-stm32mp2/drivers/stm32mp2_pmic.c b/core/arch/arm/plat-stm32mp2/drivers/stm32mp2_pmic.c
index 8fde53f3c..a61d7ffef 100644
--- a/core/arch/arm/plat-stm32mp2/drivers/stm32mp2_pmic.c
+++ b/core/arch/arm/plat-stm32mp2/drivers/stm32mp2_pmic.c
@@ -102,15 +102,15 @@ static const struct regu_dt_property prop_table[] = {
 		.prop = STPMIC2_BYPASS,
 	},
 	{
-		.name = "st,pwrctrl_enable",
+		.name = "st,pwrctrl-enable",
 		.prop = STPMIC2_PWRCTRL_EN,
 	},
 	{
-		.name = "st,pwrctrl_reset",
+		.name = "st,pwrctrl-reset",
 		.prop = STPMIC2_PWRCTRL_RS,
 	},
 	{
-		.name = "st,pwrctrl_sel",
+		.name = "st,pwrctrl-sel",
 		.prop = STPMIC2_PWRCTRL_SEL,
 	},
 	{
diff --git a/documentation/devicetree/bindings/mfd/st,stpmic2.yaml b/documentation/devicetree/bindings/mfd/st,stpmic2.yaml
index 1a2b6e19b..cf1e78889 100644
--- a/documentation/devicetree/bindings/mfd/st,stpmic2.yaml
+++ b/documentation/devicetree/bindings/mfd/st,stpmic2.yaml
@@ -108,16 +108,16 @@ properties:
             description: select VOUT4 as input source for LDO1.
             $ref: /schemas/types.yaml#/definitions/flag
 
-          st,pwrctrl_enable:
+          st,pwrctrl-enable:
             description: Enable the power control input line for the regulator.
             $ref: /schemas/types.yaml#/definitions/flag
 
-          st,pwrctrl_sel:
+          st,pwrctrl-sel:
             description: Power control input line number to use to drive
               the regulator low power state.
             $ref: /schemas/types.yaml#/definitions/uint32
 
-          st,pwrctrl_reset:
+          st,pwrctrl-reset:
             description: The regulator state is reset to it's NVMEM default
               value when exit from low power. It is disabled when the
               power control line is low.
@@ -178,8 +178,8 @@ examples:
 
           buck1 {
             regulator-name = "vdd_core";
-            st,pwrctrl_sel = <1>;
-            st,pwrctrl_enable;
+            st,pwrctrl-sel = <1>;
+            st,pwrctrl-enable;
             regulator-min-microvolt = <1200000>;
             regulator-max-microvolt = <1200000>;
             default {
-- 
2.47.2

